visionforge-solid/src/commonMain/kotlin/space/kscience/visionforge/solid/Extruded.kt

```diff
@@ -103,6 +92,24 @@ public class Extruded(
     }
 }
 
+public class ExtrudeBuilder(
+    public var shape: List<Point2D> = emptyList(),
+    public var layers: MutableList<Layer> = ArrayList(),
+    config: Config = Config()
+) : SimpleVisionPropertyContainer<Extruded>(config) {
+    public fun shape(block: Shape2DBuilder.() -> Unit) {
+        this.shape = Shape2DBuilder().apply(block).build()
+    }
+
+    public fun layer(z: Number, x: Number = 0.0, y: Number = 0.0, scale: Number = 1.0) {
+        layers.add(Layer(x.toFloat(), y.toFloat(), z.toFloat(), scale.toFloat()))
+    }
+
+    internal fun build(): Extruded = Extruded(shape, layers).apply { configure(config) }
+}
+
 @VisionBuilder
-public fun VisionContainerBuilder<Solid>.extrude(name: String? = null, action: Extruded.() -> Unit = {}): Extruded =
-    Extruded().apply(action).also { set(name, it) }
\ No newline at end of file
+public fun VisionContainerBuilder<Solid>.extruded(
```

Comment:
This method is used in other modules, for example in GDML converter. Since I do not changes, the build is probably broken. Please check that everything builds properly.

Reply:
I guess, one commit was lost with changes in GDML.